### ISSUE TITLE
Update release number to match Drupal release numbers

### ISSUE THREAD

**Issue by jenlampton** [backdrop-contrib/facetapi#10]
I'm currently working on a port of Ajax Facets that depends on `facetapi (>=7.x-1.4)`.  That means that to work with Backdrop, this should be updated to at least `facetapi (>=1.x-1.4)`. 

Unfortunately, that doesn't work because this module's stable release version is only `1.x-1.0.01`.

I don't think this module is a port of the `7.x-1.0` version of the drupal module, because that was released on 28 June 2012, which is before backdrop 1.0 was released.

The most recent version of Facet API (`7.x-1.9`) was released on 27 August 2021, which is after the most recent commit to this project, which was Dec 14, 2020. 

In short, I have no idea which version of the Drupal module this is :)

**Comment by herbdool** [comment]
The last commit from the Drupal module appears to be on 21 Dec, 2017.
